sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -469,8 +470,55 @@ def _create_table_properties(
         if clustered_by:
             properties.append(exp.Cluster(expressions=[exp.column(col) for col in clustered_by]))
 
+        if table_description:
+            properties.append(
+                exp.Property(
+                    this=exp.Identifier(this="description", quoted=False),
```

Comment:
I think this needs to be `exp.var` instead of an identifier?

Reply:
I was mimicking that sqlglot returns for a parsed statement - should I have identified the correct expression via another mechanism?

```
❯ python
Python 3.11.6 (main, Nov  2 2023, 04:39:43) [Clang 14.0.3 (clang-1403.0.22.14.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from sqlglot import parse_one
>>> parse_one("CREATE TABLE IF NOT EXISTS `test_table` (`a` int OPTIONS (description='a description'), `b` int) OPTIONS (description='test description')", dialect="bigquery")
(CREATE this: 
  (SCHEMA this: 
    (TABLE this: 
      (IDENTIFIER this: test_table, quoted: True)), expressions: 
    (COLUMNDEF this: 
      (IDENTIFIER this: a, quoted: True), kind: 
      (DATATYPE this: Type.INT, nested: False, prefix: False), constraints: 
      (COLUMNCONSTRAINT kind: 
        (PROPERTIES expressions: 
          (PROPERTY this: 
            (IDENTIFIER this: description, quoted: False), value: 
            (LITERAL this: a description, is_string: True))))), 
    (COLUMNDEF this: 
      (IDENTIFIER this: b, quoted: True), kind: 
      (DATATYPE this: Type.INT, nested: False, prefix: False))), kind: TABLE, exists: True, properties: 
  (PROPERTIES expressions: 
    (PROPERTY this: 
      (IDENTIFIER this: description, quoted: False), value: 
      (LITERAL this: test description, is_string: True))))
```

Reply:
I see - no your reasoning makes sense, maybe the way this is parsed is not fully correct as well. You can ignore my comment for the time being and I'll get back to you on this once I dig a bit deeper.